text/0016-named-tracers.md

```diff
@@ -66,22 +74,46 @@ Examples (based on existing contribution libraries from OpenTracing and OpenCens
 
 ## Internal details
 
-By providing a `TracerFactory` / `MeterFactory` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
+By providing a `TracerRegistry` / `MeterRegistry` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
 
 On an SDK level, the SpanData class and its Metrics counterpart are extended with a `getLibraryResource` function that returns the resource associated with the Tracer / Meter that created it.
 
+## Glossary of Terms
+
+#### Reporting library
+Also known as the instrumentation library or the trace/metrics reporter, this may be either a library/module/plugin provided by OpenTelemetry that instruments an existing library, a third party integration which instruments some library, or a library that has implemented the OpenTelemetry API in order to instrument itself. In any case, the reporting library is the library which provides tracing and metrics data to OpenTelemetry.
+
+examples:
+* `@opentelemetry/plugin-http`
+* `io.opentelemetry.redis`
+* `redis-client` (in this case, `redis-client` has instrumented itself with the OpenTelemetry API)
+
+#### Tracer name / Meter name
+When it is acquired from the tracer/meter registry, the tracer/meter is assigned a name and a version which is the name and version of the reporting library that acquired the tracer. In cases where a library has instrumented itself using the OpenTelemetry API, they may be the same.
+
+example: If the `http` library is being instrumented by a library with the name `io.opentelemetry.contrib.http`, then the tracer name is also `io.opentelemetry.contrib.http`. If that same `http` library has built-in instrumentation through use of the OpenTelemetry API, then the tracer name would be `http`.
+
+### Disambiguation with related terms
+
+#### Meter namespace
+This is included here because it is often confused with the meter name.
+
+This describes the component which is being instrumented. For instance, `latency` could be collected from a number of different things (http latency, disk latency, event loop latency) and a namespace ties the `latency` metric to the entity being metered.
```

Comment:
"This describes the component which is being instrumented"
->
"A meter namespace describes the component which is being instrumented"

Also: Is there another page where meter namespace is described? Would it make sense to link it?

Reply:
I was looking for other references to namespace and the spec actually seems to disagree with us in it's current state:

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics-user.md#metric-names

>Metrics names belong to a namespace by virtue of a "Named" Meter instance. A "Named" Meter refers to the requirement that every Meter instance must have an associated component label, determined statically in the code. The component label value of the associated Meter serves as its namespace, allowing the same metric name to be used in multiple libraries of code, unambiguously, within the same application.

Reply:
There is also nowhere else in oteps or spec with namespaces. This might be part of why people are so confused.

Reply:
Oh, good catch. That spec part part we should also try to fix.